app/javascript/spec/textual_summary/data/report_information_list.js

```diff
@@ -0,0 +1,14 @@
+export const reportInformationListData = {
+    items: [
+      {
+        label: 'Last Run Time',
+        value: '& 2022-12-07 02:00:00 UTC &'
+      }, 
```

Comment:
According to ESLint this file has a trailing white space on line 6 and a few other auto-fixable spacing things